great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -211,11 +209,25 @@ class GEDependencies:
         GE_DEV_DEPENDENCIES_EXCLUDED_FROM_TRACKING
     )
 
-    def __init__(self, requirements_relative_base_dir: str = "../../../") -> None:
-        self._requirements_relative_base_dir = file_relative_path(
-            __file__, requirements_relative_base_dir
-        )
-        self._dev_requirements_prefix: str = "requirements-dev"
+    DEV_REQUIREMENTS_PREFIX = "requirements-dev"
+    PRIMARY_REQUIREMENTS_FILE = "requirements.txt"
+
+    def __init__(self) -> None:
+        self._requirements_paths = self._init_requirements_paths()
```

Comment:
Since this really isn't a point of extensibility, I decided to refactor and use `pathlib` as much as possible.